packages/vscode-extension/tests/runWebviewTests.ts

```diff
@@ -150,7 +150,10 @@ export const config: Options.Testrunner = {
   // See the full list at http://mochajs.org/
   mochaOpts: {
     ui: 'tdd',
-    timeout: 120000,
+    timeout:
+      process.env.DEBUG_VSCODE_TESTS === 'true'
+        ? Number.POSITIVE_INFINITY
+        : 120000,
```

Comment:
I've added a `DEBUG_VSCODE_TESTS` variable that prevents the tests from timing out (this is a problem when you are running several tests and you are inspecting things in the debugger)